coral-schema/src/main/java/com/linkedin/coral/schema/avro/SchemaUtilities.java

```diff
@@ -618,6 +632,10 @@ static Schema makeNonNullable(Schema schema) {
   }
 
   static Schema makeNullable(Schema schema) {
+    return makeNullable(schema, false);
+  }
+
+  static Schema makeNullable(Schema schema, boolean nullAsSecond) {
```

Comment:
could you also please add a unit test in `ViewToAvroSchemaConverterTests.java` where we generate the schema from an orc table? Something similar to `testSchemaWithNullableFieldsAndDefaults` but from an orc table.
All the test tables we currently create are avro as per this method - https://github.com/linkedin/coral/blob/master/coral-schema/src/test/java/com/linkedin/coral/schema/avro/TestUtils.java#L166